content/Carla-Fabricio/newRecipeFT/New document.md

```diff
@@ -4,7 +4,11 @@ This is gonna be the best buttermilk pancake that you have ever see
 
 ***
 
-![cover](C:\Users\Arduino employee\Documents\Arduino Github\musical-pancake\content\Carla-Fabricio\images\cover.jpeg)
+
+
+![BEST BUTTERMILK PANCAKE EVER](images/recipeCover.jpg)
+
+
 
 * Toons of buttermilk
```

Comment:
It is gonna be  super healthy

Reply:
what do I need to add to make it unhealthy